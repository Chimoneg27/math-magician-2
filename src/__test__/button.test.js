import React from 'react';
import renderer from 'react-test-renderer';
import Buttons from '../components/button';

describe('Button Component Snapshot Tests', () => {
  test('display the 1 button', () => {
    const component = renderer.create(
      <Buttons clickHandler={() => {}} />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
