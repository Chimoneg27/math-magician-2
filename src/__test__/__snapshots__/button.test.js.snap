// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component Snapshot Tests display the 1 button 1`] = `
<div
  className="rows"
>
  <div
    className="row1"
  >
    <button
      onClick={[Function]}
      type="button"
    >
      AC
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      +/-
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      %
    </button>
    <button
      className="far-right"
      onClick={[Function]}
      type="button"
    >
      รท
    </button>
  </div>
  <div
    className="row1"
  >
    <button
      onClick={[Function]}
      type="button"
    >
      7
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      8
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      9
    </button>
    <button
      className="far-right"
      onClick={[Function]}
      type="button"
    >
      x
    </button>
  </div>
  <div
    className="row1"
  >
    <button
      onClick={[Function]}
      type="button"
    >
      4
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      5
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      6
    </button>
    <button
      className="far-right"
      onClick={[Function]}
      type="button"
    >
      -
    </button>
  </div>
  <div
    className="row1"
  >
    <button
      onClick={[Function]}
      type="button"
    >
      1
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      2
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      3
    </button>
    <button
      className="far-right"
      onClick={[Function]}
      type="button"
    >
      +
    </button>
  </div>
  <div
    className="bottom"
  >
    <button
      className="zero"
      onClick={[Function]}
      type="button"
    >
      0
    </button>
    <button
      className="point"
      onClick={[Function]}
      type="button"
    >
      .
    </button>
    <button
      className="far-right"
      onClick={[Function]}
      type="button"
    >
      =
    </button>
  </div>
</div>
`;

exports[`Button Component Snapshot Tests display the 2 button 1`] = `
<div
  className="rows"
>
  <div
    className="row1"
  >
    <button
      onClick={[Function]}
      type="button"
    >
      AC
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      +/-
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      %
    </button>
    <button
      className="far-right"
      onClick={[Function]}
      type="button"
    >
      รท
    </button>
  </div>
  <div
    className="row1"
  >
    <button
      onClick={[Function]}
      type="button"
    >
      7
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      8
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      9
    </button>
    <button
      className="far-right"
      onClick={[Function]}
      type="button"
    >
      x
    </button>
  </div>
  <div
    className="row1"
  >
    <button
      onClick={[Function]}
      type="button"
    >
      4
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      5
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      6
    </button>
    <button
      className="far-right"
      onClick={[Function]}
      type="button"
    >
      -
    </button>
  </div>
  <div
    className="row1"
  >
    <button
      onClick={[Function]}
      type="button"
    >
      1
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      2
    </button>
    <button
      onClick={[Function]}
      type="button"
    >
      3
    </button>
    <button
      className="far-right"
      onClick={[Function]}
      type="button"
    >
      +
    </button>
  </div>
  <div
    className="bottom"
  >
    <button
      className="zero"
      onClick={[Function]}
      type="button"
    >
      0
    </button>
    <button
      className="point"
      onClick={[Function]}
      type="button"
    >
      .
    </button>
    <button
      className="far-right"
      onClick={[Function]}
      type="button"
    >
      =
    </button>
  </div>
</div>
`;
